src/policy/fees.h

```diff
@@ -19,6 +20,8 @@
 #include <string>
 #include <vector>
 
+extern const char* FEE_ESTIMATES_FILENAME;
```

Comment:
167076739f02ddcf754200b378d504d320d0ff22:

Instead of exporting this, what about exporting a function that calculates the path?

```
fs::path FeeestPath(ArgsManager&);
```

So caller could call `FeeestPath(gArgs)` instead of `gArgs.GetDataDirNet() / FEE_ESTIMATES_FILENAME`

Reply:
Closing as resolved